src/Editor.ts

```diff
@@ -227,17 +227,37 @@ export default class Editor extends EventDispatcher {
     return this.set(new TextDocument(new Delta().insert(text), selection), source);
   }
 
-  getActive() {
-    const { selection } = this.doc;
-    let active = selection
-      ? selection[0] === selection[1]
-      ? { ...this.activeFormats, ...this.doc.getLineFormat(selection) }
-      : { ...this.doc.getFormats(selection) }
-      : {};
-    Object.values(this.modules).forEach(module => {
-      if (module.getActive) active = { ...active, ...module.getActive() };
-    });
-    return active;
+  fixupSelection(selection) {
+    const fixedSelection = selection;
+    if (fixedSelection) {
+      const selectedText = this.getText(fixedSelection);
+      if (selectedText) {
+        const trimmedText = selectedText.trim();
+        const diff = selectedText.length - trimmedText.length;
+        if (diff > 0) {
```

Comment:
If _only_ whitespace is selected, nothing should be changed. `if (diff && trimmedText.length)`

Reply:
done. Good catch.